Resources/doc/reference/form_field_definition.rst

```diff
@@ -227,16 +227,23 @@ You can easily add a new ``galleryHasMedias`` row by defining one of these optio
                 ->add('enabled')
                 ->add('name')
                 ->add('defaultFormat')
-                ->add('galleryHasMedias', 'sonata_type_collection', array(), array(
-                    'edit' => 'inline',
-                    'inline' => 'table',
-                    'sortable' => 'position',
-                    'limit' => 3
+                ->add('galleryHasMedias', 'sonata_type_collection', array(
```

Comment:
Oh he just means using FQCNs instead of type names like `sonata_type_collection`. See https://speakerdeck.com/webmozart/symfony2-forms-dos-and-donts for more details